### DISCUSSION TITLE
Configure style in the editor

### DISCUSSION THREAD

**Discussion by pabli24** [openstyles/stylus#1490] [Ideas]
This feature already exists when you install the style, so why not also in the normal editor?

example:
![configedit](https://user-images.githubusercontent.com/22481052/195450087-354681bf-99b5-4591-b0ea-c93e7f7b741d.png)

**Comment by tophf** [comment]
It doesn't make a lot of sense there. It's not that useful in the installer either, we show it there just to advertise the existence of style settings so the users know the style is configurable. The intended place to use the settings is in the Stylus popup on a matching site where you can immediately see the result of the change.
